com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -2771,6 +2709,13 @@ AnimationOnlyExportData exportData
                         return false;
                     }
 
+                    // Rotation order and rotation active must be set before setting the transform, otherwise
+                    // values may change if setting the rotation order afterwards.
+                    if (!fbxNode.GetRotationActive())
+                    {
+                        fbxNode.SetRotationOrder(FbxNode.EPivotSet.eSourcePivot, FbxEuler.EOrder.eOrderZXY);
+                        fbxNode.SetRotationActive(true);
+                    }
```

Comment:
Should SetRotationOrder/SetRotationActive be in CreateFbxNode so we set it in just one place?